### PR TITLE
Update conf files location

### PR DESCRIPTION
**Pull request by omri374**
## Change Description

This PR changes the default location of the analyzer's `conf` files to be inside the package (i.e. inside `presidio_analyzer`) so that it gets added to the wheel file that's uploaded to PyPI.

## Checklist

- [X] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [X] I have signed the CLA (if required)
- [X] My code includes unit tests
- [X] All unit tests and lint checks pass locally
- [X] My PR contains documentation updates / additions if required

### CODE REVIEW

File: presidio-analyzer/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+recursive-include conf *.yaml
```
Comment (RoeyBC) [inline presidio-analyzer/MANIFEST.in:1]: is this still needed?

File: presidio-analyzer/presidio_analyzer/conf/default.yaml [hunk 2]
```diff
@@ -33,3 +33,4 @@ ner_model_configuration:
   - PRODUCT
   - QUANTITY
   - WORK_OF_ART
+  - SOME_LABEL
```
Comment (RoeyBC) [inline presidio-analyzer/presidio_analyzer/conf/default.yaml:36]: Just making sure you wanted to add this label and keep the test

**Review by RoeyBC — APPROVED** [review]

File: presidio-analyzer/MANIFEST.in (hunk 1 above)
Comment (omri374) [reply presidio-analyzer/MANIFEST.in:1]: Good catch!

File: presidio-analyzer/presidio_analyzer/conf/default.yaml (hunk 2 above)
Comment (omri374) [reply presidio-analyzer/presidio_analyzer/conf/default.yaml:36]: Another good catch!

File: presidio-analyzer/Dockerfile.windows [hunk 3]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.9-windowsservercore
 
-ARG NLP_CONF_FILE=conf/default.yaml
+ARG NLP_CONF_FILE=presidio_analyzer/conf/transformers.yaml
```
Comment (SharonHart) [inline presidio-analyzer/Dockerfile.windows:3]: default.yaml

**Review by SharonHart — APPROVED** [review]

**Review by SharonHart — DISMISSED** [review]

**Review by RoeyBC — APPROVED** [review]

File: presidio-analyzer/Dockerfile.windows (hunk 3 above)
Comment (RoeyBC) [reply presidio-analyzer/Dockerfile.windows]: check out docker-compose-transformers.yaml too

Comment (omri374) [reply presidio-analyzer/Dockerfile.windows]: There it is supposed to be tramsformers.yaml, doesn't it?

Comment (omri374) [reply presidio-analyzer/Dockerfile.windows]: Fixed it docker-compose-transformers

**Review by RoeyBC — APPROVED** [review]

**Pull request merged**